seed/views/v3/cycles.py

```diff
@@ -0,0 +1,136 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2020, The Regents of the University of California,
+through Lawrence Berkeley National Laboratory (subject to receipt of any
+required approvals from the U.S. Department of Energy) and contributors.
+All rights reserved.  # NOQA
+:authors Paul Munday<paul@paulmunday.net> Fable Turas <fable@raintechpdx.com>
+"""
+from seed.filtersets import CycleFilterSet
+from seed.models import Cycle
+
+from seed.serializers.cycles import CycleSerializer
+from seed.utils.viewsets import SEEDOrgNoPatchOrOrgCreateModelViewSet
+
+
+class CycleViewSet(SEEDOrgNoPatchOrOrgCreateModelViewSet):
+    """API endpoint for viewing and creating cycles (time periods).
+
+        Returns::
+            {
+                'status': 'success',
+                'cycles': [
+                    {
+                        'id': Cycle`s primary key,
+                        'name': Name given to cycle,
+                        'start': Start date of cycle,
+                        'end': End date of cycle,
+                        'created': Created date of cycle,
+                        'properties_count': Count of properties in cycle,
+                        'taxlots_count': Count of tax lots in cycle,
+                        'organization': Id of organization cycle belongs to,
+                        'user': Id of user who created cycle
+                    }
+                ]
+            }
+
+
+    retrieve:
+        Return a cycle instance by pk if it is within user`s specified org.
+
+        :GET: Expects organization_id in query string.
```

Comment:
yes, for sure!

Reply:
Looks like you ended up removing the documentation instead of trying to reformat. I think this is fine since the actual params section has those details anyway.